ios/Classes/DBManager.m

```diff
@@ -27,30 +27,37 @@ @implementation DBManager
 -(instancetype)initWithDatabaseFilePath:(NSString *)dbFilePath{
     self = [super init];
     if (self) {
-        // Set the documents directory path to the documentsDirectory property.
-        NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
-        self.documentsDirectory = [paths objectAtIndex:0];
+        // Set the app directory path to the application support files property.
+        self.appDirectory = [NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES) firstObject];
 
         // Keep the database filepath
         self.databaseFilePath = dbFilePath;
 
         // Keep the database filename.
         self.databaseFilename = [dbFilePath lastPathComponent];
 
-        // Copy the database file into the documents directory if necessary.
-        [self copyDatabaseIntoDocumentsDirectory];
+        // Copy the database file into the app directory if necessary.
+        [self copyDatabaseIntoAppDirectory];
     }
     return self;
 }
 
--(void)copyDatabaseIntoDocumentsDirectory{
-    // Check if the database file exists in the documents directory.
-    NSString *destinationPath = [self.documentsDirectory stringByAppendingPathComponent:self.databaseFilename];
+-(void)copyDatabaseIntoAppDirectory{
```

Comment:
```suggestion
// Will be removed in the next major version.
-(void)copyDatabaseIntoAppDirectory{
```

Reply:
Yeah, tested it in different situations. First app opening, second app opening.

Most importantly:
Open app with current version of the lib, download file. Close the app then upgrade code with this PR. On app launch, the DB file is successfully moved to the app directory.

Reply:
Okay, great!